docs/csharp/language-reference/operators/collection-expressions.md

```diff
@@ -39,6 +39,8 @@ A *collection expression* can be converted to different collection types, includ
 - Any type with a *create* method whose parameter type is `ReadOnlySpan<T>` where there's an implicit conversion from the collection expression type to `T`.
 - Any type that supports a [collection initializer](../../programming-guide/classes-and-structs/object-and-collection-initializers.md#collection-initializers), such as <xref:System.Collections.Generic.List%601?displayProperty=nameWithType>. Usually, this requirement means the type supports <xref:System.Collections.Generic.IEnumerable%601?displayProperty=nameWithType> and there's an accessible `Add` method to add items to the collection. There must be an implicit conversion from the collection expression elements' type to the collection's element type. For spread elements, there must be an implicit conversion from the spread element's type to the collection's element type.
 
+The [Feature specification](~/_csharplang/proposals/csharp-12.0/collection-expressions.md) specifies how the compiler determines how to convert a collection expression to a target collection. The specification provides flexibility for the compiler to generate the most performant conversion to the target expression. For example, the empty collection expression, `[]`, can be realized as <xref:System.Array.Empty%60%601?displayProperty=nameWithType> if the target won't be modified after initialization. When the target is a <xref:System.Span%601?displayProperty=nameWithType> or <xref:System.ReadOnlySpan%601?displayProperty=nameWithType>, the storage may be stack allocated.
```

Comment:
Total nit. Two things:

1. Should **Feature** be capitalized?
2. As a reader about this subject, I don't necessarially care too much as to what the feature spec says things should operate. I'm more concerned about how they operate. I would suggest reversing the paragraph. Lead with how it works and follow up with more information bits related to the feature spec.
3. As a reader coming in, what's the "feature specification" anyhow? The C# language specification?